mne_bids/tests/test_write.py

```diff
@@ -110,9 +113,12 @@ def fn(fname, *args, **kwargs):
 test_convert_data = test_eegieeg_data.copy()
 test_convert_data.append(('CTF', 'testdata_ctf.ds', _read_raw_ctf))
 
-test_convertbrainvision_data = [
-    ('Persyst', 'sub-pt1_ses-02_task-monitor_acq-ecog_run-01_clip2.lay', _read_raw_persyst),  # noqa
-    ('NihonKohden', 'MB0400FU.EEG', _read_raw_nihon)
+test_converteeg_data = [
+    ('Persyst', 'BrainVision', 'sub-pt1_ses-02_task-monitor_acq-ecog_run-01_clip2.lay', _read_raw_persyst),  # noqa
+    ('NihonKohden', 'BrainVision', 'MB0400FU.EEG', _read_raw_nihon),
+    ('Persyst', 'EDF', 'sub-pt1_ses-02_task-monitor_acq-ecog_run-01_clip2.lay', _read_raw_persyst),  # noqa
+    # XXX: Idk why this one doesn't work.
+    # ('NihonKohden', 'EDF', 'MB0400FU.EEG', _read_raw_nihon),
```

Comment:
did you find out?

Reply:
No, there's some weird electrodes perhaps in that file, or something fishy going on in mne reading perhaps. Not sure how to pin it down yet

Reply:
```
>           assert_array_almost_equal(
                raw.get_data(), raw2.get_data()[:, :orig_len], decimal=6)
E           AssertionError: 
E           Arrays are not almost equal to 6 decimals
E           
E           Mismatched elements: 139199 / 139200 (100%)
E           Max absolute difference: 10.80262834
E           Max relative difference: 9.00052626
E            x: array([[-1.931642e-04, -2.970705e-04,  1.092772e-04, ...,  4.902326e-05,
E                   -6.357439e-05, -1.533205e-04],
E                  [ 2.416990e-04,  7.587873e-05,  3.805662e-04, ...,  1.666990e-04,...
E            y: array([[ 0.001253,  0.001234,  0.001271, ...,  0.001271,  0.001253,
E                    0.001253],
E                  [ 0.001289,  0.001271,  0.001308, ...,  0.001289,  0.001271,...
```

Reply:
Maybe it's something to do with:

1. Nihon reader
2. the Nihon dataset
3. EDF export not handling cals/range correctly? Although, I don't see why those are needed, cuz `get_data` should return data in the physical format, and then cal/range for EDF is just `1`.
4. Something else...